docs/recipes/browser-testing.md

```diff
@@ -7,29 +7,33 @@ An example of this is React, at least if you want to use ReactDOM.render and sim
 
 This recipe works for any library that needs a mocked browser environment.
 
-## Install jsdom
+## Install node-browser-environment
 
-Install [jsdom](https://github.com/tmpvar/jsdom).
+Install [node-browser-environment](https://github.com/lukechilds/node-browser-environment).
 
-> A JavaScript implementation of the WHATWG DOM and HTML standards, for use with node.js
+> Simulates a global browser environment using jsdom.
 
 ```
-$ npm install --save-dev jsdom
+$ npm install --save-dev node-browser-environment
```

Comment:
It's pretty easy. You just bump major on the existing one and deprecate with a message saying it's been renamed. We bump major first so the deprecation notice doesn't annoy existing users until they bump a version. I would personally do it as that module is going to be used a lot and `browser-environment` would look nicer in the import statement, but your choice ;)